doc/configuring_your_machine.md

```diff
@@ -98,34 +98,38 @@ The repo has a launch.json file that will launch the version of Python that is f
 
 ## Running Tests:
 #### Command line
-##### Windows:
   Provided your PYTHONPATH was set correctly, you can run the tests from your `<root clone>` directory.
 
-  To test the core of the CLI:
-  ```BatchFile
-  python -m unittest discover -s src/azure-cli/azure/cli/tests
-  ```
+##### Windows:
 
-  To test the command modules:
+  To run the CLI unit tests:
   ```BatchFile
-  python scripts/command_modules/test.py
+  <clone root>\scripts\run_tests.bat
```

Comment:
Recommend using the usage string from:

`run_tests [-h] [--module MODULES [MODULES ...]] [--parallel] [--live] [--test TESTS]`

That will alert people there are more options they can explore with -h.